lib/services/finamp_logs_helper.dart

```diff
@@ -62,32 +54,33 @@ class FinampLogsHelper {
   /// Write logs to a file and share the file
   Future<void> shareLogs() async {
     final tempDir = await getTemporaryDirectory();
-    final tempFile = File(path_helper.join(tempDir.path, "finamp-logs.txt"));
-    tempFile.createSync();
-
-    if (_logFileWriter != null) {
-      final basePath = (Platform.isAndroid || Platform.isIOS)
-          ? await getApplicationDocumentsDirectory()
-          : await getApplicationSupportDirectory();
-      var oldLogs =
-          File(path_helper.join(basePath.path, "finamp-logs-old.txt"));
-      var newLogs = File(path_helper.join(basePath.path, "finamp-logs.txt"));
-      if (oldLogs.existsSync()) {
-        await tempFile.writeAsBytes(await oldLogs.readAsBytes(),
-            mode: FileMode.writeOnly);
-      }
-      if (newLogs.existsSync()) {
-        await tempFile.writeAsBytes(await newLogs.readAsBytes(),
-            mode: FileMode.writeOnlyAppend);
-      }
-    } else {
-      await tempFile.writeAsString(getSanitisedLogs());
-    }
+    final zipFile = File(path_helper.join(tempDir.path, "finamp-logs.zip"));
```

Comment:
can you add the timestamp to the filename here like I suggested on Discord a while ago?